website/tfe.erb

```diff
@@ -17,6 +17,10 @@
                             <a href="/docs/providers/tfe/d/agent_pool.html">tfe_agent_pool</a>
                         </li>
 
+                        <li<%= sidebar_current("docs-datasource-tfe-ip-ranges") %>>
+                            <a href="/docs/providers/tfe/d/ip_ranges.html">tfe_agent_pool</a>
```

Comment:
```suggestion
                            <a href="/docs/providers/tfe/d/ip_ranges.html">ip_ranges</a>
```

Reply:
Whoops! Good catch.

Reply:
🤦 but mine was also not completely correct 😁